pkg/relay/relay.go

```diff
@@ -459,7 +459,8 @@ func (rs *Relay) prepareContents(submitBlockRequest *types.BuilderSubmitBlockReq
 				BlockNumber: s.Payload.Payload.Data.BlockNumber,
 				NumTx:       uint64(len(s.Payload.Payload.Data.Transactions)),
 			},
-			Timestamp: s.Payload.Payload.Data.Timestamp,
+			Timestamp:   uint64(time.Now().UnixMilli() / 1_000),
```

Comment:
You've probably already thought of this, but can you tweak the log formatter to get the precision you want?  That's the "cleaner" way.

Reply:
What do you mean by log formatter? This is not related to logging

Reply:
Oh, right.  I just assumed it was.  Nevermind then! 🙂

Reply:
Even if it was in logs I would argue that implicit precision changes are "cleaner".